docs/05-concepts/10-authentication/04-providers/06-custom-providers.md

```diff
@@ -41,7 +41,7 @@ For many authentication platforms the `userIdentifier` is the user's email, but
 
```

Comment:
Suggestion: A little bit further up on this page we have the scentence "The methods that you must implement yourself is authenticateUser and findOrCreateUser,". I don't think it is connected at all to the example that is referenced. Could you have a look at that line as well when doing this update?